gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -484,7 +518,6 @@ def unfurl_nested_annotations(
 
 def make_info_expr(
     t: hl.Table,
-    hist_prefix: str = "",
```

Comment:
495 should be dropped as well

Reply:
L495? The CTT?

Reply:
I got what you mean, old L495, the docstring.